doc/sailfish-webview-webviewexperimental.qdoc

```diff
@@ -0,0 +1,181 @@
+
+The height of the bounding box rectangle defined by the loaded content
+being displayed by the webview.
+
+See also: \l{WebView::contentRect}
+
+\section2 WebView::scrollableSize
+
+\c{QSizeF}-type read-only property.
+
+The geometric size of the content being displayed by the webview,
+in device pixels.
+
+The loaded page content is likely to fill an area much larger than the
+viewport of the webview (thus requiring the user to scroll the viewport in
+order to see content further down the page, etc).
+
+This property contains the total size of the content, in device pixels.
+
+See also: \l{WebView::contentRect}, \l{WebView::scrollableOffset}
+
+\section2 WebView::scrollableOffset
+
+\c{QPointF}-type read-only property.
+
+The (x and y pixel) offset into the scrollable size which describes
+the top-left pixel of the content being displayed by the viewport.
+
+The value of this property is effectively equal to the value of the
+x and y values of the \l{WebView::contentRect} after applying the appropriate
+\l{resolution} scale factor (to map from CSS pixel coordinates to
+device-resolution pixel coordinates).
+
+See also: \l{WebView::contentRect}, \l{WebView::scrollableSize},
+\l{WebView::resolution}
+
+\section2 WebView::resolution
+
+\c{float}-type read-only property
+
+The scaling factor which is applied to the (CSS-pixel) contentRect
+values to map to the appropriate device-resolution pixel coordinates.
+
+The value of this property is equal to the width of the device's screen
+divided by the width of the \l{WebView::contentRect}.  As such, it will update
+when the user zooms in or out (thus changing the width, in CSS pixels, of
+the contentRect), or if the page orientation changes (resulting in a change
+to landscape orientation from portrait orientation, or vice-versa).
+
+See also: \l{WebView::contentRect}
+
+\section2 WebView::horizontalScrollDecorator
+
+\l{QmlMozScrollDecorator}-type read-only property.
+
+The horizontal scroll bar.
+
+The QmlMozScrollDecorator type is available from the
+Qt5Mozilla import namespace.
+
+\section2 WebView::verticalScrollDecorator
+
+\l{QmlMozScrollDecorator}-type read-only property.
+
+The vertical scroll bar.
+
+The QmlMozScrollDecorator type is available from the
+Qt5Mozilla import namespace.
+
+\section2 WebView::painted
+
+\c{bool}-type read-only property.
+
+Whether the first paint has completed.
+
+\section2 WebView::parentId
+
+\c{int}-type read-only property.
+
+The uniqueId of the parent window (web view instance) which requested
+opening this location.
+
+\section2 WebView::uniqueId
+
+\c{int}-type read-only property.
+
+The unique identifier of this web view instance.
+
+\section1 Signals
+
+\section2 WebView::imeNotification(int state, bool open, int case, int focusChange, string type)
+
+Emitted when an input method notification is received by the view.
+
+For example, if the user gives focus to an input field, an input method
+notification will be emitted.
```

Comment:
Is it possible/useful to know about the parameters?

Reply:
Well, I'm not sure how useful it is, but I've added some documentation about those now, based on a rudimentary reading of the gecko-dev source..

Reply:
Ah great; I definitely think it's helpful. It at least removes confusion for the reader.